### ISSUE TITLE
Managing Share2Fedi via a systemd unit.

### ISSUE THREAD

**Issue by Zergy** [kytta/share2fedi#53]
Hello,

I manage to install Share2fedi on my own instance this morning. For some reason it keep crashing and restarting when I tried to start it with PM2, but maybe it's my fault, but as it work fine with the "node" command directly, I keep it this way.

I created a little systemd unit to start and stop it, it could be nice to add it to the files of the project, so interacting with it will be easier than launching a command by hand.
For your info, I made a copy of the "mastodon-web" unit as a base and modified, so, I guess it's not perfect.

I'm sorry if it's not the right place for that, as it's not a issue but more an addition proposal, I'm still a noob with github. :/

[share2fedi.service.txt](https://github.com/kytta/share2fedi/files/12777255/share2fedi.service.txt)

**Comment by kytta** [comment]
> I created a little systemd unit to start and stop it, it could be nice to add it to the files of the project, so interacting with it will be easier than launching a command by hand.

Good idea! I am not a fan of putting too many files into the source repository, but this addition is helpful. I need time to figure out where to put it...

> For your info, I made a copy of the "mastodon-web" unit as a base and modified, so, I guess it's not perfect.

It does look quite good, actually! I have learned that systemd has sandboxing :D

> I'm sorry if it's not the right place for that, as it's not a issue but more an addition proposal, I'm still a noob with github. :/

This certainly is a good place; otherwise, you could've also opened a pull request — all fine by me! Don't worry, there's plenty of time to learn it all :) Welcome to the world of open-source!
